cmd/release-controller/sync.go

```diff
@@ -299,10 +299,23 @@ func (c *Controller) syncPending(release *releasecontroller.Release, pendingTags
 			rewriteValue := tag.Annotations[releasecontroller.ReleaseAnnotationRewrite]
 			if len(rewriteValue) == 0 {
 				klog.V(2).Infof("Rewriting pending release %s", tag.Name)
-				isi, err := c.imageClient.ImageStreamImages(release.Source.Namespace).Get(context.TODO(), fmt.Sprintf("%s@%s", release.Source.Name, id), metav1.GetOptions{})
+				var isi *imagev1.ImageStreamImage
+				isi, err = c.imageClient.ImageStreamImages(release.Source.Namespace).Get(context.TODO(), fmt.Sprintf("%s@%s", release.Source.Name, id), metav1.GetOptions{})
 				if err != nil {
 					return err
 				}
+				// Handle manifest list based releases...
+				if len(isi.Image.DockerImageManifests) > 0 {
```

Comment:
This if statement is unnecessary since we're just looping over the array.

Reply:
Removed